.travis.yml

```diff
@@ -20,25 +20,22 @@ jobs:
     - stage: test
       name: Test and Coverage
       install:
-        # Install Anaconda
+        # Install Micromamba
         # We do this conditionally because it saves us some downloading if the
         # version is the same.
-        - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
-        - bash miniconda.sh -b -p $HOME/miniconda
-        - export PATH="$HOME/miniconda/bin:$PATH"
-        - hash -r
-        - conda config --set always_yes yes --set changeps1 no
-        - conda update -q conda
+        - wget -qO- https://micromamba.snakepit.net/api/micromamba/linux-64/latest | tar -xvj bin/micromamba
+        - ./bin/micromamba shell init -s bash -p ~/micromamba
+        - source ~/.bashrc
         # Useful for debugging any issues with conda
-        - conda info -a
-
+        - micromamba info 
         # Update python version in environment
         - sed -i -E 's/(python=)(.*)/\1'$TRAVIS_PYTHON_VERSION'/' ./environment.yml
         # Create conda environment
-        - conda env create -n radis-env -f environment.yml
+        - micromamba install xtensor -c conda-forge
```

Comment:
Add `-y` argument, in the end, to  select yes automatically when `y/n` is prompted

Reply:
Great!